RELEASES.md

```diff
@@ -884,4 +884,9 @@
 
 11.13.4 2021-09-20
 -----------------
-  - fix: SMC-4161 - user permissions not cleaned up on user deletion
\ No newline at end of file
+  - fix: SMC-4161 - user permissions not cleaned up on user deletion
```

Comment:
please configure IDE to add newline at end of *.md files. We use editorconfig for that. It prevents old lines being changed when appending to file